cob_bringup/robots/cob3-9.xml

```diff
@@ -157,6 +157,7 @@
 		</include>
 		<!--include file="$(find cob_bringup)/drivers/image_flip_nodelet.launch" >
```

Comment:
I just kept the arguments as they were used for the real robot...there they are commented out anyway....I don't know why `start_manager` was set to `false` for cob3-2 and cob3-9...but I did not want to change that....